packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue

```diff
@@ -0,0 +1,410 @@
+<template>
+    <div
+        ref="cookieBanner"
+        :class="[$style['c-cookieBanner'], $style['c-cookieBanner-overlay'], { [$style['c-cookieBanner--hidden']]: hideBanner }]"
+        attr="data-cookie-consent-overlay"
+        :aria-hidden="isHidden">
```

Comment:
Do we need a `hideBanner` and an `isHidden`?

Reply:
We do not need both